src/actions/user.actions.js

```diff
@@ -91,9 +92,23 @@ const update = (
   };
 };
 
+const resetPwd = (email) => {
```

Comment:
I'm not a fan of abbreviations or misleading names in JS language because JS is verbose (not as ruby but verbose anyway😂), and when a project increase the amount of code is more difficult searching a function, variable, etc, because, as for example this case, I have to search `password` and `pwd`. Anyway, let me know what you think. _you can dismiss this comment_

Reply:
Ok I'll change that